go.mod

```diff
@@ -3,9 +3,13 @@ module github.com/istio-ecosystem/admiral-sharding-manager
 go 1.22.4
 
 require (
-	github.com/istio-ecosystem/admiral-api v1.0.2
+	github.com/google/go-cmp v0.6.0
+	github.com/google/uuid v1.6.0
+	github.com/istio-ecosystem/admiral-api v1.0.3-0.20240604180405-9243983d25e2
```

Comment:
Created a new release for admiral-api https://github.com/istio-ecosystem/admiral-api/releases/tag/v1.1.0

Reply:
thanks @nirvanagit , updated the version